### PR TITLE
Add note about generating keys from the WordPress API #192

### PR DESCRIPTION
**Pull request by rstormsf**
Add note about generating keys from the WordPress API
#192 close

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,6 +100,7 @@ In the environment files inside the `group_vars` directory, `wordpress_sites` is
   * `db_password` - database password
   * `db_host` - database hostname
   * `domain_current_site` (required for multisite)
+  * `generateme` - salt keys to generate from [wordpress_api](https://api.wordpress.org/secret-key/1.1/salt/)
```
Comment (retlehs) [inline README.md]: this is confusing. `generateme` is a value

could probably just remove this change to the README

**Comment by retlehs** [comment]
thanks, just some minor nitpicks:

* commit message should be along the lines of "Add note about generating keys from the WordPress API"
* this PR description itself should contain more info than just "#192 close"
* PR title should also reflect the commit message

i also think the change made to README should be dropped. see inline note above

**Comment by rstormsf** [comment]
@retlehs Sorry about that

**Comment by swalkinshaw** [comment]
Thanks!

**Pull request merged**